spec/system/admin_badges_spec.rb

```diff
@@ -63,6 +63,39 @@
     end
   end
 
+  context "when updating a badge" do
+    it "can upload an image for the badge" do
+      badges_page.visit_page(Badge::Autobiographer)
+      form.choose_conditional("upload-image")
+
+      attach_file(File.absolute_path(file_from_fixtures("logo.jpg"))) do
+        form.field("image_url").find(".image-upload-controls .btn").click
+      end
+
+      expect(form.field("image_url")).to have_css(".btn-danger")
+      form.submit
```

Comment:
Just a note here but I'm wondering why the form is not a concern of the `badges_page` page object. It feels like we could encapsulate the logic in there since that is the reason why page objects exists. Something like this: 

```
badges_page
  .visit_page(Badge::Autobiographer)
  .upload_image("logo.jpg")
  .submit_form
```